core/auth/signer.go

```diff
@@ -12,24 +12,26 @@ import (
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
-type signer struct {
+type LocalBlobRequestSigner struct {
```

Comment:
What other kinds of signer do we want to add in the future?

Reply:
A remote signer? Beyond the scope of this PR though.